### DISCUSSION TITLE
`GenImageSimplexNoise()`, similar to `GenImagePerlinNoise()`

### DISCUSSION THREAD

**Discussion by DreepyYunky** [raysan5/raylib#4028] [Ideas]
It'll be cool to have Simplex Noise, also written by Ken Perlin and may be similar in use to `GenImagePerlinNoise()`

**Comment by raysan5** [comment]
@DreepyYunky Afaik, `GenImagePerlinNoise()` is actually using simplex noise 3... you can check  `stb_perlin.h`.

**Reply by DreepyYunky** [reply]
Oh, Maybe it'll be better to name it `GenImageSimplexNoise` then.

**Reply by raysan5** [reply]
Afaik, Simplex is also Perlin, right? In any case, no plans to rename the function.
